system-test/spanner.ts

```diff
@@ -1317,9 +1784,28 @@ describe('Spanner', () => {
       );
       const exists = await backup1.exists();
       assert.strictEqual(exists, true);
+    };
+
+    it('GOOGLE_STANDARD_SQL should have completed a backup', async () => {
+      await completedBackup(
+        googleSqlBackup1,
+        googleSqlBackup1Name,
+        googleSqlDatabase1
+      );
+    });
+
+    it('POSTGRESQL should have completed a backup', async function () {
+      if (IS_EMULATOR_ENABLED) {
```

Comment:
nit: here and elsewhere, I think you can remove this, as backup tests should be disabled for both dialects on the emulator